docs/antrea-network-policy.md

```diff
@@ -647,6 +647,25 @@ The rules are logged in the following format:
     2021/06/24 23:56:41.346165 AntreaPolicyEgressRule AntreaNetworkPolicy:default/test-anp Drop 44900 10.10.1.65 35402 10.0.0.5 80 TCP 60 [3 packets in 1.011379442s]
 ```
 
+Kubernetes Network Policies can also be audited using Antrea logging to the same file
+(`/var/log/antrea/networkpolicy/np.log`). Set the Namespace Annotations to
+`policy.antrea.io/enable-np-logging: "true"`, then all the rules of Kubernetes
```

Comment:
not sure if this has been discussed before: policy may mean many things, wonder if we should use `networkpolicy.antrea.io/enable-logging: "true"`?

Reply:
it was only discussed to be specific as `xxx.antrea.io/xxx`, sure `networkpolicy.antrea.io/enable-logging: "true"` sounds more accurate here.

Reply:
`networkpolicy.antrea.io/enable-logging: "true"` sounds good to me.